src/cborencoder.c

```diff
@@ -41,7 +41,7 @@ void cbor_encoder_init(CborEncoder *encoder, uint8_t *buffer, size_t size, int f
     encoder->flags = flags;
 }
 
-static inline void put16(void *where, uint16_t v)
+CBOR_INLINE_API void put16(void *where, uint16_t v)
```

Comment:
This function isn't API, so I'd rather not have it using the "CBOR_*API" macros.

What was the issue? My guess: MSVC 2012 in C mode does not understand "inline". Correct?